### PR TITLE
June 2020 Refresh

### PR DESCRIPTION
**Pull request by codingbandit**
This PR adds in Exercise 6, deployments to Azure using GitHub actions. It also adds Key Vault and App Configuration resources, as well as resolves the following issues: #67 , #61, #52, #43

### CODE REVIEW

File: Hands-on lab/HOL step-by-step - Modern cloud apps.md [hunk 1]
```diff
@@ -142,7 +152,7 @@ In this exercise, you will provision a website via the Azure **Web App + SQL** t
 
     ![On the SQL Database blade, in the left pane, Overview is selected. In the right pane, under Essentials, the Connection strings (Show database connection strings) link is circled.](images/Hands-onlabstep-by-step-Moderncloudappsimages/media/image23.png "SQL Database blade")
 
-5. On the **Database connection strings** blade, select and copy the **ADO.NET** connection string. Then, save it in **Notepad** for use later, being sure to replace the placeholders with your username and password with **demouser** and **demo@pass123**, respectively.
+5. On the **Database connection strings** blade, select and copy the **ADO.NET** connection string. Then, save it in **Notepad** for use later, being sure to replace the password placeholder with **demo@pass123**
 
     ![In the Database connection strings blade, the ADO.NET connection string is circled.](images/Hands-onlabstep-by-step-Moderncloudappsimages/media/image24.png "Database connection strings blade")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Modern cloud apps.md:158]: Hello!  Can you check line 163-165 for me please?  Instruction 7 and alt-text says, .... select Set server firewall link at the top.  The image shows Show firewall settings circled.  Is this right?

File: Hands-on lab/HOL step-by-step - Modern cloud apps.md [hunk 2]
```diff
@@ -229,87 +305,138 @@ In this exercise, you will provision a website via the Azure **Web App + SQL** t
 
     ![In the App Service blade, under Settings, select Configuration link.](media/2019-04-19-16-38-54.png "Configuration link")
 
-5. Add a new **Application setting** with the following values:
+5. Locate **Connection Strings** section below **Application Settings**.
 
-   - Key: `AzureQueueConnectionString`
+    ![The Connection Strings section for the App Service blade displays.](images/Hands-onlabstep-by-step-Moderncloudappsimages/media/image41.png "Connection Strings section")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Modern cloud apps.md:310]: Check the image formatting please.  The image isn't showing in View, I only see the alt-text on the page.

**Comment by codingbandit** [comment]
Great catches @DawnmarieDesJardins! I've fixed those issues and the respective files are updated.

File: Hands-on lab/HOL step-by-step - Modern cloud apps.md [hunk 3]
```diff
@@ -1095,7 +1354,7 @@ In this exercise, you will configure an Azure AD Business to Consumer (B2C) inst
 15. A browser tab/window will open that looks like the following screenshot.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Modern cloud apps.md:1354]: Hello!  Can you please check the instructions in lines 1313 - 1318.  They don't make sense to me (could just be me though).

File: Hands-on lab/HOL step-by-step - Modern cloud apps.md [hunk 4]
```diff
@@ -1183,13 +1442,13 @@ To enable profile editing on your application, you will need to create a profile
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Modern cloud apps.md:1442]: Please check the alt-text in line 1441.  Is "Return claim attributes selected blade" right?

File: Hands-on lab/HOL step-by-step - Modern cloud apps.md [hunk 5]
```diff
@@ -1721,7 +2048,7 @@ Contoso wants to automate the process of generating receipts in PDF format and a
 
     ![A sample Contoso Sports League PDF receipt is displayed.](media/2019-04-15-16-15-06.png "Sample PDF receipt")
 
-### Task 2: Create an Azure Logic App to Process Orders
+### Task 3: Create an Azure Logic App to Process Orders
 
 Without writing any code, you can automate business processes more easily and quickly when you create and run workflows with Azure Logic Apps. Logic Apps provide a way to simplify and implement scalable integrations and workflows in the cloud. It provides a visual designer to model and automate your process as a series of steps known as a workflow. There are [many connectors](https://docs.microsoft.com/en-us/azure/connectors/apis-list) across the cloud and on-premises to quickly integrate across services and protocols.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Modern cloud apps.md:2054]: Lines 2090 - 2092 - Please check.  Instructions day select Azure Queues but the image looks like Service Bus is selected.

File: Hands-on lab/HOL step-by-step - Modern cloud apps.md [hunk 6]
```diff
@@ -2157,6 +2484,377 @@ The advantages of using Logic Apps include the following:
 
     ![The Workflow diagram begins with Recurrence, then flows to Execute stored procedure, then to Condition. The Condition fields are as follows: Object Name, ReturnCode; Relationship, is greater than; Value, 0. Below the Workflow diagram is an If Yes box, with a workflow that begins wtih Execute stored procedure 2, and flows to forEach email. There is also an If No, Do Nothing box.](images/Hands-onlabstep-by-step-Moderncloudappsimages/media/image305.png "Workflow diagram")
 
+## Exercise 6: Automate deployments using GitHub actions
+
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Modern cloud apps.md:2488]: Please add a Duration:  X minutes

File: Hands-on lab/HOL step-by-step - Modern cloud apps.md [hunk 7]
```diff
@@ -33,15 +33,39 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/in
   - [Help references](#help-references)
   - [Exercise 1: Proof of concept deployment](#exercise-1-proof-of-concept-deployment)
     - [Task 1: Deploy the e-commerce website, SQL Database, and storage](#task-1-deploy-the-e-commerce-website-sql-database-and-storage)
+      - [Subtask 1: Configure SQL Database Firewall and Retrieve Connection String](#subtask-1-configure-sql-database-firewall-and-retrieve-connection-string)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Modern cloud apps.md:36]: Hello!  Please set the TOC to ignore the subtasks.

**Pull request merged**